src/Middleware.ts

```diff
@@ -2,7 +2,7 @@ import DispatchFunction from './DispatchFunction';
 import ActionFunction from './ActionFunction';
 
 interface Middleware {
-    (next: DispatchFunction, action: ActionFunction, actionType: string, args: IArguments): void;
+    (next: DispatchFunction, action: ActionFunction, actionType: string, args: IArguments, options?: any): void;
```

Comment:
I had the same exact thought when I went home. I will update the other packages as well when i am done with this